### ISSUE TITLE
ERROR in Debug Failure. False expression.

### ISSUE THREAD

**Issue by Slanfan** [dzonatan/angular2-linky#17]
```
ERROR in node_modules/ngx-linky/lib/linky.module.d.ts:1:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class.

This likely means that the library (ngx-linky) which declares LinkyModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.

1 export declare class LinkyModule {
```
@angular/core ~9.1.6

**Comment by Slanfan** [comment]
Got it to work by changing the export declaration but guessing this will change if I would reinstall it again.

**Comment by dzonatan** [comment]
Fixed with `v3.0.0`
